tests/containers/helm.pm

```diff
@@ -14,14 +14,37 @@ use version_utils qw(is_sle);
 use registration qw(add_suseconnect_product get_addon_fullname);
 use mmapi 'get_current_job_id';
 
+
+sub install_kubectl {
+    my $version = get_var('PUBLICCLOUD_KUBECTL_VERSION', 'v1.22.2');
+    assert_script_run("curl -LO https://dl.k8s.io/release/$version/bin/linux/amd64/kubectl");
+    assert_script_run("chmod a+x kubectl");
+    script_run("rm /usr/bin/kubectl");
+    assert_script_run("mv kubectl /usr/bin/kubectl");
+}
+
+sub install_aws_cli {
+    assert_script_run("pip3 install awscli");
+    record_info('EC2', script_output('aws --version'));
+}
+
 sub run {
     my ($self) = @_;
 
     $self->select_serial_terminal;
     $self->{deployment_name} = "apache-" . get_current_job_id();
     my $chart = "bitnami/apache";
 
-    zypper_call("in -y jq aws-cli helm kubernetes-client");
+    if (is_sle) {
+        add_suseconnect_product(get_addon_fullname('phub'));
+        zypper_call("in -y jq helm python3-pip");
+        install_aws_cli();
+        install_kubectl();
+    }
+    else {
+        zypper_call("in -y jq aws-cli helm kubernetes-client");
+    }
```

Comment:
Forgot to ask you in the previous PR, sorry! Why do we need `jq` as it is not yet used anywhere ?

Reply:
It's needed by aws get-called-identify call
`> aws sts get-caller-identity | jq -r '.Account'`